DEVELOPMENT.md

```diff
@@ -61,29 +61,32 @@ hatch run all:test
 
 ##### Manual Installation
 
-1. Install `deadline[gui]` and `blender-qt-stylesheet` packages to `~/DeadlineCloudSubmitter/Submitters/Blender/python/modules`.
-  - For Windows: `pip install deadline[gui] blender-qt-stylesheet -t %USERPROFILE%\DeadlineCloudSubmitter\Submitters\Blender\python\modules`
-  - For Unix: `pip install deadline[gui] blender-qt-stylesheet -t ~/DeadlineCloudSubmitter/Submitters/Blender/python/modules`
-
-  Currently, Python 3.10 is the Python version used by the latest Blender release.
-
-2. Install addon in Blender
-
-  Copy the contents of `src/deadline/blender_submitter/addons` to `~/DeadlineCloudSubmitter/Submitters/Blender/python/addons`
-
-3. Add a script directory in Blender by "Edit" > "Preferences" > "File Paths" > "Script Directories"
-  * Windows: `%USERPROFILE%\DeadlineCloudSubmitter\Submitters\Blender\python`
-  * Linux: `~/DeadlineCloudSubmitter/Submitters/Blender/python`
-
-  Or run this script from Blender
-
-  ```
-  import bpy
-  import os
-  bpy.ops.preferences.script_directory_add(directory=os.path.expanduser(os.path.normpath('~/DeadlineCloudSubmitter/Submitters/Blender/python')))
-  ```
-
-4. Restart Blender - changes to the script directory won't take effect until Blender has been restarted.
+These instructions make the following assumptions:
+  * You have a `git clone` of this repository locally
+  * You have `pip` available in your terminal (or are using Blender's `pip`)
+
+1. Install dependencies to `~/DeadlineCloudSubmitter/Submitters/Blender/python/modules`
```

Comment:
`~/DeadlineCloudSubmitter/Submitters/Blender` -- where does this directory come from? 

I'm a person that's just done a `git clone` of this repository. How did I get this directory? Are we assuming that someone has run the Deadline Cloud submitter installer first?

Reply:
It gets created by the command

Reply:
Ahh. That was not clear to me :-)